app/src/main/java/com/sopt/kakaobank/presentation/MainActivity.kt

```diff
@@ -14,6 +16,46 @@ class MainActivity : BindingActivity<ActivityMainBinding>(R.layout.activity_main
     override fun initView() {
         // 로직 작성
         // exampleObserve()
+        initHomeFragment()
+        clickBottomNavigation()
```

Comment:
```suggestion
        initBottomNavClickListener()
```
- **initXXX()** : 초기화 함수 이름
    - **init[View]ClickListener** : 클릭 리스너 설정